### ISSUE TITLE
No node types beyond NamedThing

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/kg-phenio#58]
We really would like to have Biolink node categories in KG-Phenio, but right now even the types KGX is capable of assigning categories to aren't getting them:
```
~/kg-phenio/data/merged$ grep MONDO: merged-kg_nodes.tsv | cut -f 2 | sort | uniq
biolink:NamedThing
```

So there should be *some* categories for nodes.

Barring KGX assignment, these can be mapped by prefix.

**Comment by matentzn** [comment]
FYI in the old monarch ontology, we merged the biolink model itself into phenio and ran a custom sparql query:

https://github.com/monarch-initiative/monarch-ontology/blob/master/Makefile#L88

Not sure this should be the strategy moving forward.

**Comment by caufieldjh** [comment]
I see some pros and cons to that approach - it would be consistent with the other preprocessing steps to include biolink categories in phenio, but this leaves it unclear which version of the model is being used. The potential to use a modified Monarch-specific version of the model has also come up more than once, so ideally we should have a process in which the desired model version is just specified and applied (and perhaps this is more efficiently done over the graph rather than the OWL).

We see similar issues with KG-OBO: https://github.com/Knowledge-Graph-Hub/kg-obo/issues/111

**Comment by matentzn** [comment]
Maybe the best way is to manually curate top level mappings in an sssom file (bl:phenotype exactMatch UPHENO:123) merge that into phenio and then propagate the bl categories using sparql update dynamically..

**Comment by caufieldjh** [comment]
So the current plan is to attack this on multiple fronts:
* For now, mimic what was done re: merging Biolink for monarch-ontologies in https://github.com/monarch-initiative/monarch-ontology/blob/master/Makefile#L88. This will be in a new PR on PHENIO.
* In the longer term, have KGX assign BL categories at transform time - see https://github.com/biolink/kgx/issues/378 and https://github.com/biolink/kgx/pull/385
* As a backup option, provide SSSOM maps from BL classes to specific CURIEs - this is what [BioPortal-to-KGX](https://github.com/ncbo/BioPortal-to-KGX) does; see https://github.com/ncbo/BioPortal-to-KGX/tree/main/mappings. This should happen upstream for the sake of resilience

**Comment by matentzn** [comment]
This sounds perfect to my ears as well!

**Comment by caufieldjh** [comment]
Fixed in build [20220930](https://kg-hub.berkeleybop.io/kg-phenio/20220930/)
